about.html

```diff
@@ -0,0 +1,177 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>About</title>
+    <link href='https://unpkg.com/boxicons@2.1.4/css/boxicons.min.css' rel='stylesheet'>
+    <link rel="stylesheet" href="assets/style/about.css">
+    <link rel="stylesheet" href="assets/style/style.css">
+    <script src="./assets/scripts/menu.js" defer></script>
+    <link rel="stylesheet" href="assets/style/aboutMediaquery.css">
+    <link rel="stylesheet" href="assets/style/mainMediaqueries.css">
+
+</head>
+
+<body>
+    <div class="top-contact">
+        <ul>
+            <li>English</li>
+            <li>My Page</li>
+            <li>Logout</li>
+        </ul>
+    </div>
+    <header class="main-header">
+        <nav>
+            <div class="menu-logo">
+                <div class="menu-con">
+                    <i class='bx bx-menu'></i>
+                </div>
+
+                <div class="logo">
+                    <img src="assets/images/logo.png" alt="">
+                </div>
+
+            </div>
+
+            <ul>
+                <a  href="/">
+                    <li>Home</li>
+                </a>
+                <a class="active-link" href="about.html">
+                    <li>About</li>
+                </a>
+                <a href="#">
+                    <li>Join</li>
+                </a>
+                <a href="#">
+                    <li>Sponsor</li>
+                </a>
+                <a href="#">
+                    <li>News</li>
+                </a>
+            </ul>
+        </nav>
+
+        <div class="mobile-menu">
+            <div class="close-btn-container">
+                <i class='bx bx-x'></i>
+            </div>
+            <ul>
+                <a class="" href="/">
+                    <li>Home</li>
+                </a>
+                <a href="about.html">
+                    <li>About</li>
+                </a>
+                <a href="#">
+                    <li>Join</li>
+                </a>
+                <a href="#">
+                    <li>Sponsor</li>
+                </a>
+                <a href="#">
+                    <li>News</li>
+                </a>
+            </ul>
+        </div>
+    </header>
+
+    <div class="about-hero">
```

Comment:
Consider using a `section` tag as a wrapper here and in other places where you are beginning a section of a page with `div` in order to make your code more semantically correct.